cluster-up/cluster/kind-k8s-sriov-1.17.0/config_sriov.sh

```diff
@@ -225,7 +225,9 @@ function move_sriov_pfs_netns_to_node {
   mkdir -p /var/run/netns/
   ln -sf /proc/$pid/ns/net "/var/run/netns/$node"
 
-  local -r sriov_pfs=( /sys/class/net/*/device/sriov_numvfs )
+  local -r sriov_pfs=( $(find /sys/class/net/*/device/sriov_numvfs) )
+  [ "${#sriov_pfs[@]}" -eq 0 ] && echo "FATAL: Could not find available sriov PFs" > /dev/stderr && return 1
```

Comment:
I do not know where you saw this `> /dev/stderr` used, I know about a simple `>&2`.
And I am not clear why `&&` was used after, `echo` usually succeeds, but a more direct approach is to use `;` in case some poor author copy/paste this pattern and places something that can fail instead of the `echo`.

Note that for printing to stderr, you can also set a helper function: `echoerr() { printf "%s\n" "$*" >&2; }` that is better than `echo` as it does not require an exec.

Reply:
`false && echo bla; echo foo`
will print foo, unless i use '( )' on the right part
&& is pretty wide used i think, and is straight forward and clear.
wdyt?

I think the echoerr is overkill, unless you insist.

Reply:
You used `&& return 1` and that should be `; return 1`.

The function is more readable and avoids adding the redirection each time. The fact it is faster is less important.
I will not insist in the function.

Reply:
```
$ false && echo bla; return 1
bash: return: can only `return' from a function or sourced script
```
You can't it will execute it anyhow, unless i use 'false && (echo bla; return 1)`

Reply:
Yes, I see now. My mistake.